pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete_modules.adoc

```diff
@@ -0,0 +1,17 @@
+[id="delete_modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
+
+.Prerequisites
+
+//* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Select the check box of the module you want to publish.
```

Comment:
shouldn't this be "the module you want to delete." ?

Reply:
that's me being distracted... ;-)